### PR TITLE
Improve member-ordering 

### LINKED ISSUE #80: Improve member-ordering
We should add ability to sort members alphabetically inside one group.
Also should add support for Angular view children/content children annotations.

### PR DESCRIPTION
**Pull request by incendial**
Changes:
- fix multiple same group members warning message
- add support for angular view children and content children decorators
- add support for alphabetize option

Closes #80

### CODE REVIEW

File: lib/src/rules/member_ordering.dart [hunk 1]
```diff
@@ -236,12 +283,24 @@ class _Annotation {
...
+  static const viewChild =
+      _Annotation._('ViewChild', _MembersGroup.angularViewChild);
+  static const viewChildren =
+      _Annotation._('ViewChildren', _MembersGroup.angularViewChild);
+  static const contentChild =
+      _Annotation._('ContentChild', _MembersGroup.angularContentChild);
+  static const contentChildren =
+      _Annotation._('ContentChildren', _MembersGroup.angularContentChild);
 
   static List<_Annotation> annotations = [
```
Comment (dkrutskikh) [inline lib/src/rules/member_ordering.dart:295]: @incendial can we make private this variable ?

File: lib/src/rules/member_ordering.dart [hunk 2]
```diff
@@ -202,6 +244,9 @@ class _MembersGroup {
   static const angularOutputs = _MembersGroup._('angular_outputs');
   static const angularHostBindings = _MembersGroup._('angular_host_bindings');
   static const angularHostListeners = _MembersGroup._('angular_host_listeners');
+  static const angularViewChild = _MembersGroup._('angular_view_children');
+  static const angularContentChild =
+      _MembersGroup._('angular_content_children');
 
   static List<_MembersGroup> groupsOrder = [
```
Comment (dkrutskikh) [inline lib/src/rules/member_ordering.dart:251]: @incendial can we make private this variable ?

File: doc/rules/member_ordering.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ member-ordering
 
 Enforces member ordering.
 
-The value for order may be an array consisting of the following strings:
+The value for `order` may be an array consisting of the following strings:
```
Comment (Anrock) [inline doc/rules/member_ordering.md:11]: I guess it should be mentioned that following is the default order. Or if it isn't - add a list with default ordering.

File: lib/src/rules/member_ordering.dart (hunk 2 above)
Comment (incendial) [reply lib/src/rules/member_ordering.dart]: Fixed

File: lib/src/rules/member_ordering.dart (hunk 1 above)
Comment (incendial) [reply lib/src/rules/member_ordering.dart]: Fixed

File: doc/rules/member_ordering.md (hunk 3 above)
Comment (incendial) [reply doc/rules/member_ordering.md]: I have added a description, that its a default order

**Pull request merged**